### DISCUSSION TITLE
Get number of pending notifications

### DISCUSSION THREAD

**Discussion by ilvacca** [isaacrlevin/presencelight#832] [Ideas]
Hi! I love this app. But I was wondering: is there a way to get the number of Teams pending notifications? Does the Graph API expose this information?

The "old" Teams app used to expose this information in the logs, but now the new version doesn't seem to log anything.

Thank you for your effort :)

**Comment by isaacrlevin** [comment]
Not sure honestly, as this is something I have never looked into. Maybe this doc is a good starting point
https://learn.microsoft.com/en-us/graph/teams-send-activityfeednotifications?tabs=http
